packages/compiled-assets/README.md

```diff
@@ -52,18 +65,30 @@ const app = express();
 app.use('/build/', assets.handler());
 ```
 
-To include a bundle on your page, you can use the `compiledScriptTag` or `compiledScriptPath` functions. The name of the bundle passed to this function is the filename of your bundle within the `scripts` directory.
+To include a bundle on your page, you can use the `compiledScriptTag` or `compiledScriptPath` functions. The name of the bundle passed to this function is the filename of your bundle within the `scripts` directory. You can use `compiledScriptPreloadTags` to get a list of tags that should be preloaded in the `<head>` of your HTML document, or `compiledScriptPreloadPaths` to get the paths.
```

Comment:
What's the utility of `compiledScriptPreloadPaths`? That is, when would that return anything? I thought preloads were mostly useful when chunking (or more generally, ESM imports on the client) were involved.

Reply:
Ah -- I forgot that by default it would create a large bundle for a compiled script that includes imports.

Reply:
Changed in 088f05771de6887836a96364aeaee9a82fee5021.